src/plugins/links/server/plugin.ts

```diff
@@ -38,6 +41,44 @@ export class LinksServerPlugin implements Plugin<object, object> {
       },
     });
 
+    plugins.embeddable.registerEmbeddableFactory({
```

Comment:
This change is not necessary for this PR. It will be necessary to register links embeddable on the server side when we [handle references at the server level](https://github.com/elastic/kibana/issues/192758). Fixed in [14319b](https://github.com/elastic/kibana/pull/193067/commits/14319b2069667a6c4a9b598d931d65a0d7bb75c4)